developer/configuration.md

```diff
@@ -56,6 +56,10 @@ Reaction supports sending mail over SMTP; the `MAIL_URL` environment variable sh
 
 The Email dashboard provides a UI for quick configuration of the email server as well.
 
+### SKIP_FIXTURES
+
+Pass in `SKIP_FIXTURES` to `reaction` to prevent overwriting custom Shop data with fixture data from `/private/data/Shops.json`.
```

Comment:
Using "Pass in" here seems confusing if we are talking about setting environment variables.